s3.py

```diff
@@ -111,9 +127,9 @@ def cover_image_url(cls, data_source, identifier, filename=None,
         return root + "%s/%s/%s" % tuple(args)
 
     @classmethod
-    def feed_url(cls, filename, extension='.opds', open_access=True):
+    def feed_url(cls, filename, extension='.xml', open_access=True):
         """The path to the hosted file for an OPDS feed with the given filename"""
-        root = cls.content_root(open_access)
+        root = cls.static_feed_root(open_access)
```

Comment:
It was a temporary measure while the dev S3 bucket was being created, though I never actually intended to put anything in the content bucket.